server/model.js

```diff
@@ -1,16 +1,16 @@
-const database = require('./database.js');
-
-const getImages = function(params){
-  var query = "select * from images WHERE id = ?;";
-  return new Promise( (resolve, reject) => {
-    database.query(query, params, function(err,results){
-      if (err){
-        reject(err);
-      } else{
-        resolve(results);
-      }
-    })
-  })
-}
-
-module.exports = getImages;
+const DatabaseConnection = require('./database.js');
+
+const getImages = function (params) {
+  const query = 'select * from images WHERE id = ?;';
```

Comment:
I don't think you need explicit ; at the end of query.

Reply:
I looked into this and you're right. You are correct; this isn't necessary for the query statement. I read up on this and saw conflicting ideologies about this. I think I'm going to keep the explicit semi colon in the hopes of possibly avoiding future bugs occurring from not terminating statement.